src/lib.rs

```diff
@@ -220,6 +220,10 @@ pub struct Join {
     /// Clear data directory created by a previous node run
     #[structopt(long = "clear-data")]
     clear_data: bool,
+
+    /// Use this flag to pass --skip-igd to the node binary, which will disable port forwarding.
+    #[structopt(long)]
+    disable_port_forwarding: bool,
```

Comment:
I'm wondering about the naming here.

Should it not be `disable_skipping_igd` ? and if not... does that mean that the naming of `skip-igd` is bad?

Reply:
Hi Josh,

My understanding is that IGD is software-based port forwarding, and we have this error message:
```
                    "You can disable port forwarding by supplying --skip-igd. Without port\n\
                    forwarding, your machine must be publicly reachable by the given\n\
                    --public-addr. If your machine is not publicly reachable, you may have to\n\
                    adjust your router settings to either:\n\
```
So 'disable port forwarding' seemed a bit more understandable to me than 'skip IGD' (I personally had never heard the term IGD until I started doing this work). The term 'disable skipping' seems a touch clunky to me, since they're both terms about removing something.